HW3/VladykaYura/hw2.py

```diff
@@ -0,0 +1,21 @@
+
+user_number = input("Enter your number:")
+
+user_number_list = list(map(int, user_number))
+
+user_number_sum = sum(user_number_list)
+print("Number sum :",user_number_sum)
+
```

Comment:
look at this version of code:
number = int(input("Enter four digit number: "))
num_1 = number // 1000
num_2 = (number // 100) % 10
num_3 = (number // 10) % 10
num_4 = number % 10
product = num_1 * num_2 * num_3 * num_4
print(product)